functions/src/bills/types.ts

```diff
@@ -53,6 +53,297 @@ export const BillContent = Record({
  * to appear in results when sorting by that value. */
 export const MISSING_TIMESTAMP = Timestamp.fromMillis(0)
 
+export const TOPICS_BY_CATEGORY = {
```

Comment:
Should we just make every string lower case? I feel like the capitalization is somewhat inconsistent, and I wonder if that will affect future usage of topics.

Reply:
Yeah, definitely agree with getting consistent capitalization here - the categories and topics were only officially finalized after this PR went up, so these are technically a WIP - but the final version still has some inconsistency.

I'll check in with Matt V tonight on how he wants to handle this - I'd prefer to just set the capitalization/formatting as a product decision so we don't have to think about this in code. Either way, I'll update this before the end of the night.

Reply:
The final decision we came to at hack night is to only capitalize the first letter of each topic - I've updated the topics to reflect this change.